sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -491,7 +491,14 @@ class JDBCSuite extends QueryTest
   }
 
   test("Partitioning via JDBCPartitioningInfo API") {
-    val df = spark.read.jdbc(urlWithUserAndPass, "TEST.PEOPLE", "THEID", 0, 4, 3, new Properties())
+    val df = spark.read.jdbc(urlWithUserAndPass,
+      "TEST.PEOPLE",
+      "THEID",
+      "0",
+      "4",
+      3,
+      new Properties()
+    )
```

Comment:
Let's change like the following.
```scala
val df =
  spark.read.jdbc(urlWithUserAndPass, "TEST.PEOPLE", "THEID", 0, 4, 3, new Properties())
```

Reply:
Ping, @MrDLontheway , for the above comment.

Reply:
thanks for reminding